### DISCUSSION TITLE
[Column Chart] Customize legend

### DISCUSSION THREAD

**Discussion by theoanca** [IgniteUI/igniteui-angular#12112] [General]
## Question  

Is there a way to customize the legend labels for the column and bar charts?

igniteui-angular version: 14.0.14
igniteui-angular-charts version: 14.1.1
browser: Chrome

**Comment by anserban** [comment]
@MayaKirova, @dkamburov, would this be possible to have or is there a workaround?

**Comment by dkamburov** [comment]
Checkout the following topics:

- https://www.infragistics.com/products/ignite-ui-angular/angular/components/charts/features/chart-data-tooltip#angular-data-tooltip-styling
- https://www.infragistics.com/products/ignite-ui-angular/angular/components/charts/features/chart-data-legend#angular-data-legend-styling

**Comment by theoanca** [comment]
Thank you for your answer.
